mon-pix/app/templates/terms-of-service-pole-emploi.hbs

```diff
@@ -1,49 +1,48 @@
-{{! template-lint-disable no-implicit-this no-unknown-arguments-for-builtin-components }}
 {{page-title (t "pages.terms-of-service-pole-emploi.title")}}
 <div class="terms-of-service-page">
   <div class="terms-of-service-form">
 
     <div class="terms-of-service-form__logo">
-      <a href={{homeUrl}}>
+      <a href={{this.homeUrl}}>
```

Comment:
J'ai pas trouvé de test qui couvrent cette partie, y'a celui-ci qui a l'air `mon-pix/tests/acceptance/terms-of-service_test.js` mais l'user se connecte par email.
Pas pu tester fonctionnellement, j'ai une 409 :-(

Reply:
Je vais voir pour ajouter un test pour ce cas de figure.

Reply:
J'ai tenté un test mais peu concluant. Tu as pu trouver une solution pour ce 409 ?